### DISCUSSION TITLE
Add JULIA.Net to Julia Interop?

### DISCUSSION THREAD

**Discussion by HyperSphereStudio** [HyperSphereStudio/JULIA.Net#10] [General]
This library still requires a lot of testing before I would suggest such a thing but I just wanted to make this discussion to hear other people's thoughts who also use this library.

Adding it to Julia Interop would help others find this faster who are in need of C# <-> Julia

**Comment by BambOoxX** [comment]
For me it would be awesome in terms of visibility for such an interface. However, I do not know how one can propose a package / project for an organization...
